config/test.exs

```diff
@@ -188,3 +188,5 @@ config :exvcr,
 config :echo_boy, port: 4100
 
 config :ex_bitmex, domain: "testnet.bitmex.com"
+
+config :ex_unit, assert_receive_timeout: 200
```

Comment:
😎 if this is the problem. Thanks for looking into it.

How do you feel about making this configurable as an env var with a default?

Reply:
This test is failing on CI only. I can extract it once I can confirm this the cause.